### PR TITLE
Added eslint to npm test and added eslintignore for coverage

### PR DESCRIPTION
**Pull request by troy0820**
- This will add `eslint . --ext .js`  to test along with `mocha .`
- Added `.eslintignore` to not eslint `coverage/`
- Deleted `pretest` script

### CODE REVIEW

**Comment by lynnaloo** [comment]
I already beat you to some of this

**Comment by troy0820** [comment]
When I saw it I was like 😬 Lol.

**Comment by troy0820** [comment]
@lynnaloo I rebased it with your changes to add my changes

File: package.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "description": "Oran.js JavaScript by Orange Cats",
   "main": "index.js",
   "scripts": {
-	 "pretest": "npm run lint",
-    "test": "gulp test",
+    "test": "eslint . --ext .js && gulp test",
+    "start": "node app/index.js",
```
Comment (lynnaloo) [inline package.json]: this should be `node app/app.js`

Comment (troy0820) [reply package.json]: Yes it should, I honestly didn't even check that.  I didn't add that but I can take care of it.

**Pull request merged**